packages/bits/src/lib/wizard-v2/wizard-vertical/wizard-vertical.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="nui-step" *ngFor="let step of steps; let i = index; let isLast = last">
-    <wizard-step-header class="nui-wizard-vertical-header"
+    <nui-wizard-step-header class="nui-wizard-vertical-header"
```

Comment:
according to BEM, it should be `nui-wizard-vertical__header`

Reply:
It was decided to stop working on the vertical wizard since there is no use case and we have no specs for it. We'll be reworking this type of wizard once there is a demand and clear requirements. I removed this component from public apis to not confuse anyone.